src/Stratis.Features.Collateral/CollateralFeature.cs

```diff
@@ -41,6 +42,15 @@ public static class FullNodeBuilderCollateralFeatureExtension
         // Both Cirrus Peg and Cirrus Miner calls this.
         public static IFullNodeBuilder CheckForPoAMembersCollateral(this IFullNodeBuilder fullNodeBuilder, bool isMiner)
         {
+            if (!isMiner)
+            {
+                // Remove the PoAHeaderSignatureRule if this is not a miner as CirrusD has no concept of the federation and
```

Comment:
Perhaps update the comment as follows: "This rule is only required to ensure that the mining nodes don't mine on top of bad blocks. A consensus error will prevent that from happening."